openfisca_france/model/prestations/minima_sociaux/cmu.py

```diff
@@ -275,7 +276,7 @@ def function(self, simulation, period):
 
         res = cmu_br_i_par[CHEF] + cmu_br_i_par[PART] + forfait_logement
 
-        res += 12 * (aspa + ass + asi + af + cf + asf + paje_clca)
+        res += 12 * (aspa + ass + asi + af + cf + asf + paje_clca + paje_prepare)
```

Comment:
I know this is not the subject of your commit but I am concerned by the number 12 here. Is it by law or is it a left over from the times we did not deal with infra annual values ?